lnbits/extensions/lndhub/views_api.py

```diff
@@ -116,19 +117,25 @@ async def lndhub_balance(
 
 @lndhub_ext.get("/ext/gettxs")
 async def lndhub_gettxs(
-    wallet: WalletTypeInfo = Depends(check_wallet), limit: int = Query(0, ge=0, lt=200)
+    wallet: WalletTypeInfo = Depends(check_wallet),
+    limit: int = Query(20, ge=1, le=20),
+    offset: int = Query(0, ge=0),
 ):
     for payment in await get_payments(
         wallet_id=wallet.wallet.id,
         complete=False,
         pending=True,
         outgoing=True,
         incoming=False,
+        limit=limit,
+        offset=offset,
         exclude_uncheckable=True,
     ):
         await payment.set_pending(
             (await WALLET.get_payment_status(payment.checking_id)).pending
         )
+        await asyncio.sleep(0.1)
```

Comment:
Hey @callebtc, I noticed this line. Does that mean it'll take 2 seconds to get 20 payments from LndHub? Is there a reason to keep this?

Reply:
Yes, I added this to prevent DoS in an earlier version but it could be omitted now.